src/redux/index.ts

```diff
@@ -8,33 +8,47 @@ import { LogStreamState, Message } from "./logStream/logStream.types";
 import logStreamReducer from "./logStream/logStream.reducer";
 import api from "../flux/api";
 import { handleNewLog } from "./logStream/logStream.actions";
+import { GlobalState } from "./global/global.types";
+import globalReducer from "./global/global.reducer";
+import thunk from "redux-thunk";
+import { handleConnectionStatus } from "./global/global.actions";
 
 export type State = {
   flowState: FlowState;
   settingsState: SettingsState;
   logStreamState: LogStreamState;
+  globalState: GlobalState;
 };
 
 const rootReducer = combineReducers({
   flowState: flowReducer,
   settingsState: settingsReducer,
   logStreamState: logStreamReducer,
+  globalState: globalReducer,
 });
 
-const store = createStore(rootReducer, composeWithDevTools());
+const middleware = [thunk];
 
-function handleLogConnectionStatus(status: string, message: string) {}
+const store = createStore(
+  rootReducer,
+  composeWithDevTools(applyMiddleware(...middleware))
```

Comment:
Is there a reason for adding another variable for middleware?

Why not just `applyMiddleware(thunk)`?

Reply:
Good question :D